src/pages/taskQueue/index.tsx

```diff
@@ -15,14 +15,16 @@ import {
 import { getTaskQueueRoute, getAllHostsRoute } from "constants/routes";
```

Comment:
I think the page is crashing for me when I visit the `/task-queue` route, which is linked via the nav bar

Reply:
It looks like the lg table is not performant on larger tables. I am adding virtualization support in #2205

Reply:
I'd be curious to know if the original tanstack table has the same problem 🤔 When I was working on the build visibility demo, I actually had to use the original tanstack table because the LG table was too slow...

Reply:
It does I tried updating the example table to render 10k rows and it would crash repeatedly https://codesandbox.io/p/devbox/optimistic-gianmarco-vpj64c